templates/web/pages/bmviii_apply.tmpl

```diff
@@ -0,0 +1,469 @@
+                                            <option value="Computer Engineering" selected>Computer Engineering</option>
+                                            {{ else }}
+                                            <option value="Computer Engineering">Computer Engineering</option>
+                                            {{ end }}
+
+                                            {{ if eq .FormRefill.Major "Data Science" }}
+                                            <option value="Data Science" selected>Data Science</option>
+                                            {{ else }}
+                                            <option value="Data Science">Data Science</option>
+                                            {{ end }}
+
+                                            {{ if eq .FormRefill.Major "Software Engineering" }}
+                                            <option value="Software Engineering" selected>Software Engineering</option>
+                                            {{ else }}
+                                            <option value="Software Engineering">Software Engineering</option>
+                                            {{ end }}
+
+                                            {{ if eq .FormRefill.Major "Mathematics" }}
+                                            <option value="Mathematics" selected>Mathematics</option>
+                                            {{ else }}
+                                            <option value="Mathematics">Mathematics</option>
+                                            {{ end }}
+
+                                            {{ if eq .FormRefill.Major "Other" }}
+                                            <option value="Other" selected>Other</option>
+                                            {{ else }}
+                                            <option value="Other">Other</option>
+                                            {{ end }}
+                                    </select>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+					</div>
+
+                    {{ if eq .FormRefill.Major "Other" }}
+					<div id="other-major-input" class="field">
+                        <label for="other-major" class="label">
+                            <p class="bmviii-app-font" style="margin-bottom: 0px">
+                                Other Major
+                            </p>
+                        </label>
+                        <div class="field-body">
+                            <div class="field">
+                                <div class="control">
+                                    <input type="text" id="other-major" name="other-major" style="background-color: #F0F0F0" class="input" placeholder="Major not on list" minlength="7" maxlength="32" value="{{ .FormRefill.OtherMajor }}" required>
```

Comment:
I'd go through and double check these min/max lengths, like this min length seems overly restrictive

Reply:
Definitely tend towards being less restrictive rather than more

Reply:
Sounds good. I think I'm going to use the same restrictions as last year